src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -30,7 +30,7 @@ import { slickGridDataItemColumnValueWithNoData, textFormatter } from 'sql/base/
 @Component({
 	selector: 'modelview-table',
 	template: `
-		<div #table style="height:100%;" [style.font-size]="fontSize" [style.width]="width"></div>
+		<div #table style="height:100%;" [style.font-size]="fontSize" [style.width]="width" tabindex="-1"></div>
```

Comment:
wouldn't this change this for any table? for example schema compare result table - we don't want tab there too? :(

Reply:
This isn't affecting the schema compare result table because of the checkbox focus thing in #6821, so tabbing into the table still works there.

Reply:
I think all other tables would need this too because the cells aren't actionable. The DeclarativeTable has actionable cells, but that isn't impacted by this since it's a different component.

Reply:
schema compare is using this together with checkbox plugin, right? I am not sure i understand your previous comments, could you please explain how will this not affect schema compare?

Reply:
When tabbing into the schema compare table, the focus goes to the first checkbox in the first row of the table. The behavior of tabbing into the schema compare table doesn't change with this change.
![schemaCompareTab](https://user-images.githubusercontent.com/31145923/67046860-f3cd3600-f0e5-11e9-8757-b7de46fe6cfc.gif)

Reply:
We can merge this and as per recent conversation looks like as per Acessibility team tabbing should not work inside any table, only row keys.

Reply:
@kisantia does this change any behavior in other places we use this component, like Dashboard, Query Editor or Profiler?  The user's will still be able to use the keyboard to focus into these tables and navigate around?

Reply:
Yes users can still use the keyboard to navigate to and around tables